docs/modules/sm.md

```diff
@@ -0,0 +1,2 @@
+# sm
+sm is a reimplementation of Nintendo's service manager. It allows Atmosphère to add or remove process handle limits, add new services, intercept service calls. This allows high-level intercepting of Horizon OS functionality.
```

Comment:
Document the `AtmosphereInstallMitm` and `AtmosphereUninstallMitm` IPC extensions, along with making it clear that it differs from stock `sm` in that it automatically raises session limits and reimplements the `sm:h` bug.